engines/stark/ui/menu/saveloadmenu.cpp

```diff
@@ -265,10 +265,12 @@ void SaveDataWidget::loadSaveDataElements() {
 		}
 
 		// Obtain the thumbnail
-		Graphics::Surface *thumb = metadata.readGameScreenThumbnail(&stream);
-		_texture->update(thumb);
-		thumb->free();
-		delete thumb;
+		if (metadata.version >= 9) {
+			Graphics::Surface *thumb = metadata.readGameScreenThumbnail(&stream);
+			_texture->update(thumb);
+			thumb->free();
+			delete thumb;
+		}
```

Comment:
I don't have save data older than 9. Could you please check it for me?

Reply:
It's working, thanks!

This is my collection of old saved games in case you need it: https://filenurse.com/download/361b5da097c32696eff053a7b026ad7b.html

A lot of them were made back when some entity save handlers were missing, so they don't load all the state, but oh well...